### ISSUE TITLE
Custom Data Storage

### ISSUE THREAD

**Issue by Vitael7** [OnlyNoobDied/GadgetsMenu#816]
Upgrade the API to feed players data, like unlocked cosmetics and pets from another plugin. Just like custom economy storage.

**Comment by OnlyNoobDied** [comment]
You're talking about adding your own cosmetics into GadgetsMenu menu and also in Mystery Box loot?

**Comment by Vitael7** [comment]
No, instead of mysql or file storage i want to use custom plugin to store the data that contains information about which player has which cosmetics. Just like the Custom Economy Storage feature.

**Comment by yapzhenyie** [comment]
Sorry, no API for custom data storage for player. It's quite difficult to let developer fill in all the data correctly.
